audio/libid3tag/Portfile

```diff
@@ -1,22 +1,25 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem	1.0
+PortGroup       github 1.0
 
 name		libid3tag
-version		0.15.1b
+version		0.15.1c
 revision	2
 categories	audio
 license		GPL-2+
 maintainers	nomaintainer
 description	id3 tag manipulation library
-long_description	libid3tag is a library for reading and \
-	writing ID3 tags, bothID3v1 \
-	and the various versions of ID3v2.
+long_description	libid3tag is a library for reading and writing ID3 tags, bothID3v1 and the various versions of ID3v2.
 platforms	darwin freebsd
 homepage	http://www.underbit.com/products/mad/
-master_sites	ftp://ftp.mars.org/pub/mpeg/ sourceforge:mad
-checksums	sha1    4d867e8a8436e73cd7762fe0e85958e35f1e4306 \
-		rmd160  31a69b8ad7684aefdb675acc8ebf89bd6f432095
+
+github.setup    azakkerman libid3tag 0.15.1c
+checksums           sha1    9dcb0eb17c9c8e52c2a145073956f96720cbcf9f \
```

Comment:
Thanks for this @azakkerman - I think you can remove the `sha1` checksum. The recommended ones are `rmd160`, `sha256` and `size`, so lets just keep those.

Reply:
Updated